scripts/cooper/run_mitochondria_segmentation.py

```diff
@@ -28,6 +33,10 @@ def main():
         "--force", action="store_true",
         help="Whether to over-write already present segmentation results."
     )
+    parser.add_argument(
+        "--mask_path", help="The filepath to a tif file with a mask that will be used to restrict the segmentation."
+        "Can also be a directory with tifs if the filestructure matches input_path."
+    )
```

Comment:
The scripts now support passing a mask. The segmentation will be restricted to the inside of it (e.g. to just segment mitos / vesicles in one synaptic compartment)